<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dock_Grid</class>
 <widget class="QDockWidget" name="Dock_Grid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="configure-grid">
    <normaloff/>
   </iconset>
  </property>
  <property name="windowTitle">
   <string>Configure Grid</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>216</width>
      <height>232</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="check_enable">
       <property name="text">
        <string>Enable Grid</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spin_size">
         <property name="suffix">
          <string>px</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="combo_shape">
         <item>
          <property name="text">
           <string>Square</string>
          </property>
          <property name="icon">
           <iconset theme="grid-square">
            <normaloff/>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Triangle 1</string>
          </property>
          <property name="icon">
           <iconset theme="grid-triangle-h">
            <normaloff/>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Triangle 2</string>
          </property>
          <property name="icon">
           <iconset theme="grid-triangle-v">
            <normaloff/>
           </iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Shape</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Origin</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spin_x">
          <property name="minimum">
           <double>-8192.000000000000000</double>
          </property>
          <property name="maximum">
           <double>8192.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spin_y">
          <property name="minimum">
           <double>-8192.000000000000000</double>
          </property>
          <property name="maximum">
           <double>8192.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="button_reset">
          <property name="toolTip">
           <string>Reset grid position</string>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="button_move">
          <property name="text">
           <string>Move...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spin_x</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Dock_Grid</receiver>
   <slot>position_spin_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_y</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Dock_Grid</receiver>
   <slot>position_spin_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>position_spin_changed()</slot>
 </slots>
</ui>
